modules/data-entry/src/main/frontend/src/dataHomepage/NewFormDialog.jsx

```diff
@@ -276,99 +354,51 @@ function NewFormDialog(props) {
-                      "(" + acceptableSubjectTypes.map((condition) => `n.'requiredSubjectTypes'${condition}`).join(" OR ") + ")"
-                    );
-                  }
-
-                  if (conditions.length > 0) {
-                    sql += "WHERE " + conditions.join(" AND ");
-                  }
-
-                  sql += " order by n.'title'";
-
-                  url.searchParams.set("query", sql);
-                  url.searchParams.set("limit", query.pageSize);
-                  url.searchParams.set("offset", query.page*query.pageSize);
-                  return fetchWithReLogin(globalLoginDisplay, url)
-                    .then(response => response.json())
-                    .then(result => {
-                      return {
-                        data: result["rows"],
-                        page: Math.trunc(result["offset"]/result["limit"]),
-                        totalCount: result["totalrows"],
-                      }}
-                    )
-                }}
-                options={{
-                  search: true,
-                  searchAutoFocus: true,
-                  header: false,
-                  actionsColumnIndex: -1,
-                  addRowPosition: 'first',
-                  pageSize: pageSize,
-                  rowStyle: rowData => ({
-                    // /* It doesn't seem possible to alter the className from here */
-                    backgroundColor: (selectedQuestionnaire?.["jcr:uuid"] === rowData["jcr:uuid"]) ? theme.palette.grey["200"] : theme.palette.background.default,
-                    // // grey out subjects that have already reached maxPerSubject
-                    color: ((relatedForms?.length && (selectedSubject || currentSubject) && (relatedForms.filter((i) => (i["@name"] == rowData["@name"])).length >= (+(rowData?.["maxPerSubject"]) || undefined)))
+                  },
+                  { accessorKey: 'description' }
+                ]}
+                data={data}
+                muiTableBodyRowProps={({ row }) => ({
+                  sx: {
+                    cursor: 'pointer',
+                  },
+                  onClick: () => { !isFetching && setSelectedQuestionnaire(row.original);
+                                   setError(false);
+                                   row.toggleSelected();
+                                 },
+                  selected: row.original["jcr:uuid"] === selectedQuestionnaire?.["jcr:uuid"],
+                })}
+                muiTableBodyCellProps={({ cell }) => ({
+                  sx: {
+                    // grey out subjects that have already reached maxPerSubject
+                    color: ((relatedForms?.length && (selectedSubject || currentSubject) && (relatedForms.filter((i) => (i["@name"] == cell.row.original["@name"])).length >= (+(cell.row.original?.["maxPerSubject"]) || undefined)))
                     ? theme.palette.grey["500"]
                     : theme.palette.grey["900"]
```

Comment:
These should be replaced by 
```suggestion
                    ? theme.palette.text.disabled
                    : theme.palette.text.primary
```